go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/guilhem/rss-issues-action
 
-go 1.18
+go 1.21
```

Comment:
We have to update docker version too :)

Reply:
Applied with https://github.com/guilhem/rss-issues-action/pull/43/commits/5acc916071f4df1783e40bff9bc14ba43974b313!